src/Components/MovieList/index.tsx

```diff
@@ -16,24 +18,29 @@ import * as Selectors from './selectors'
 
 const MovieList: React.FC = () => {
     const dispatch = useDispatch()
+    const location = useLocation()
+    const searchParams = new URLSearchParams(location.search)
+
+    const sortBy = searchParams.get('sortBy') || 'genres'
+    const sortOrder = searchParams.get('sortOrder') || 'desc'
+    const { genre, searchQuery } =
+        useParams<{ genre: string; searchQuery: string }>()
+    const searchBy = searchParams.get('searchBy') || 'title'
+    const limit = searchParams.get('limit') || '12'
```

Comment:
I think that `'12'` will be better to put to constant

Reply:
Done